components/sidebar/sidebar_channel/sidebar_group_channel/sidebar_group_channel.tsx

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {Channel} from 'mattermost-redux/types/channels';
+import {PreferenceType} from 'mattermost-redux/types/preferences';
+
+import {trackEvent} from 'actions/diagnostics_actions';
+import {browserHistory} from 'utils/browser_history';
+import Constants from 'utils/constants';
+
+import SidebarChannelLink from 'components/sidebar/sidebar_channel/sidebar_channel_link';
+
+type Props = {
+    channel: Channel;
+    currentTeamName: string;
+    currentUserId: string;
+    redirectChannel: string;
+    active: boolean;
+    membersCount: number;
+    actions: {
+        savePreferences: (userId: string, preferences: PreferenceType[]) => Promise<{data: boolean}>;
+    };
+};
+
+type State = {
+
+};
+
+export default class SidebarGroupChannel extends React.PureComponent<Props, State> {
+    handleLeaveChannel = (callback: () => void) => {
+        const id = this.props.channel.id;
+        const category = Constants.Preferences.CATEGORY_GROUP_CHANNEL_SHOW;
+
+        const currentUserId = this.props.currentUserId;
+        this.props.actions.savePreferences(currentUserId, [{user_id: currentUserId, category, name: id, value: 'false'}]).then(callback);
+
+        trackEvent('ui', 'ui_direct_channel_x_button_clicked');
```

Comment:
Just double checking, is this supposed to be 'ui_**direct**_channel_x_button_clicked' instead of **group**?

Reply:
It matches what's in the original sidebar, so I kept it that way. We can change it up if you'd like.

Reply:
Seems like it should be **group** given it's in the 'sidebar_**group**_channel'.